File: PRACTICA/Java/irrgarten/MonsterSquare.java
package irrgarten;

public class MonsterSquare {
    private int row;
    private int col;
    private Monster monster;

    public MonsterSquare(int row, int col, Monster monster) {
        this.row = row;
        this.col = col;
        this.monster = monster;
    }

    public Monster get(){
        return monster;
    }

    public void set(int row, int col, Monster monster){
        this.row = row;
        this.col = col;
        this.monster = monster;
    }
}

File: PRACTICA/Ruby/Irrgarten/monster_square.rb
class Monster_square
  def initialize(row, col, monster)
    @row = row
    @col = col
    @monster = monster
  end

  def get()
    return @monster
  end

  def set(row, col, monster)
    @row = row
    @col = col
    @monster = monster
  end
end